### PR TITLE
AArch64: Fix off-by-one in exception handler

### PR DESCRIPTION
**Pull request by ostannard**
This caused register values to be printed left-shifted by 4, with the least significant digit repeated.

### CODE REVIEW

**Review by keith-packard — APPROVED** [review]

**Comment by ostannard** [comment]
Hi @keith-packard, is there anything else I need to do to get this merged?

**Comment by keith-packard** [comment]
Nope, I'm just on holiday and not responding quickly.

**Comment by keith-packard** [comment]
oh, getting things rebased to current main always helps a bunch -- picolibc CI doesn't speculatively merge before running tests, so the only way to get accurate testing is to have things based from main before merging. I've done that here and will merge sometime after tests pass.

**Pull request merged**
